src/Medidata.ZipkinTracer.Core/ZipkinClient.cs

```diff
@@ -20,11 +17,14 @@ public class ZipkinClient : ITracerClient
         private string requestName;
         private ITraceProvider traceProvider;
         private ILog logger;
+        private List<string> internalDomainList;
 
         public ZipkinClient(ITraceProvider tracerProvider, string requestName, ILog logger) : this(tracerProvider, requestName, logger, new ZipkinConfig(), new SpanCollectorBuilder()) { }
 
         public ZipkinClient(ITraceProvider traceProvider, string requestName, ILog logger, IZipkinConfig zipkinConfig, ISpanCollectorBuilder spanCollectorBuilder)
         {
+            internalDomainList = zipkinConfig.GetInternalDomainList();
```

Comment:
@kenyamat I updated the documents in the readme and configs in respect with the new config. Also renamed the config to notToBeDisplayedDomainList 3cbcbf6
Please check if this approach is ok.
Thanks.